charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -498,7 +507,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
-[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-1.25.0...HEAD
+[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-dashboards-1.26.0...HEAD
```

Comment:
Nit: Would you mind also fixing the other `opensearch-` to `opensearch-dashboards-` since you caught this error 😅

Reply:
Sure, I will update it

Reply:
Thanks!